R/constants.R

```diff
@@ -139,14 +148,45 @@ aglu.NO_AGLU_REGIONS <- "Taiwan"
 # Define GCAM category name of fertilizer
 aglu.FERT_NAME <- "N fertilizer"
 
+# Average Wood Density kg/m^3 for mass conversion
+aglu.AVG_WOOD_DENSITY_KGM3 <- 500
+
+# Average Agriculture Density for mass conversion
+# no idea of units, just hardcoded in as 1 in old DS
```

Comment:
Units are kg per cubic meter.
For source, we could cite lots of things; here's one option: http://www.engineeringtoolbox.com/wood-density-d_40.html
As far as I know nobody's ever done a weighted average wood density across all tree species that are commercially logged; the number 500 was just aiming to be towards the middle of the species that are produced.